<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PriceForm</class>
 <widget class="QWidget" name="PriceForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>Входные данные</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>281</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Страна абонента</string>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>261</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>38</y>
      <width>181</width>
      <height>273</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="durationLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="dayComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Понедельник</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Вторник</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Среда</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Четверг</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Пятница</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Суббота</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Воскресенье</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Продолжительность беседы
 в минутах:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>День недели:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="applyPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>481</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать стоимость разговора</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="resetPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>501</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Результат</string>
   </property>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
